planner/core/plan_cost.go

```diff
@@ -62,6 +62,11 @@ func (p *basePhysicalPlan) GetPlanCost(taskType property.TaskType, costFlag uint
 	return p.planCost, nil
 }
 
+// GetNetworkCost calculates the cost of the plan in network data transfer.
+func (p *basePhysicalPlan) GetNetworkCost() float64 {
+	return 0.0
```

Comment:
Why returning a constant 0.0?

Reply:
There are only 4 type of PHysicalPlans that will actually communicate with tikv, so the default implement for other plans should be 0.